automation/jleechanorg_pr_automation/jleechanorg_pr_monitor.py

```
@@ -1122,10 +1926,61 @@ def _count_codex_automation_comments(self, comments: List[Dict]) -> int:
+                    count += 1
+            elif workflow_type == "fix_comment":
+                # Fix-comment workflow uses dedicated markers for queued runs + completion.
+                if self.FIX_COMMENT_RUN_MARKER_PREFIX in body or self.FIX_COMMENT_MARKER_PREFIX in body:
+                    count += 1
+            elif workflow_type == "fixpr":
+                # FixPR workflow uses a dedicated marker in its queued comment.
+                if self.FIXPR_MARKER_PREFIX in body:
+                    count += 1
+            else:
+                # Fallback: count all automation comments
+                if (
```

Ruff (0.14.10)

Use `elif` instead of `else` then `if`, to reduce indentation

Convert to `elif`

(PLR5501)